src/app/utility/meter-grouping/meter-grouping.component.ts

```diff
@@ -21,7 +21,7 @@ export class MeterGroupingComponent implements OnInit {
     meterGroupIds: Array<string>
   }>;
 
-  groupMenuOpen: number;
+  /*groupMenuOpen: number;*/
```

Comment:
If this or the other commented code isn't going to be needed in the future, then we should just delete this code instead of commenting